content/kots-cli/restore/_index.md

```diff
@@ -15,11 +15,15 @@ kubectl kots restore [flags]
 
 - _Provide `[flags]` according to the table below_
 
-| Flag              | Type   | Description                                                                     |
-| :---------------- | ------ | ------------------------------------------------------------------------------- |
-| `-h, --help`      |        | help for restore                                                                |
-| `--from-backup`   | string | the name of the backup to restore from _(required)_                             |
-| `-o, --output`    | string | output format (currently supported: json) _(defaults to plain text if not set)_ |
+| Flag                        | Type   | Description                                                                                   |
+| :-------------------------- | ------ | --------------------------------------------------------------------------------------------- |
+| `-h, --help`                |        | help for restore                                                                              |
+| `--from-backup`             | string | the name of the backup to restore from _(required)_                                           |
+| `--exclude-admin-console`   | bool   | exclude restoring the admin console and only restore the application(s) _(defaults to false)_ |
+| `--exclude-apps`            | bool   | exclude restoring the application(s) and only restore the admin console _(defaults to false)_ |
+| `--wait-for-apps`           | bool   | wait for all applications to be restored _(defaults to true)_                                 |
+| `--velero-namespace`        | string | the namespace in which Velero is installed                                                    |
```

Comment:
thanks @ajp-io, yeah i didn't add that flag because the user wouldn't need to pass it except for minimal rbac installations, and the UI already includes that. but i guess we should document it anyway. i update the description to reflect that.